posixlib/src/main/scala/scala/scalanative/posix/sys/stat.scala

```diff
@@ -132,3 +132,40 @@ object stat {
   def S_IXOTH: mode_t = extern
 
 }
+
+object statOps {
```

Comment:
As struct field access like `buf._8` can not be marked as `@deprecated`, I think wrapping struct in an implicit class is a feasible solution that minimizes the effect of `stat` changes in the future.